deeplake/core/chunk_engine.py

```diff
@@ -1463,13 +1467,13 @@ def _check_rechunk(self, chunk: BaseChunk, chunk_row: int):
             chunk.num_data_bytes < RANDOM_MINIMAL_CHUNK_SIZE
             and self.max_chunk_size > RANDOM_MINIMAL_CHUNK_SIZE
         ):
-            self._try_merge_with_neighbor_and_split(chunk=chunk, row=chunk_row)
+            return self._try_merge_with_neighbor_and_split(chunk=chunk, row=chunk_row)
```

Comment:
same here, it doesn't return anything, is it to just short circuit? if else? won't help much elif is already ignored

Reply:
this was part of a different way to check if rechunk happened, forgot to revert it.